lumigator/python/mzai/backend/backend/services/jobs.py

```diff
@@ -179,27 +122,60 @@ def create_evaluation_job(self, request: JobCreate) -> JobResponse:
         if request.system_prompt is None and not request.model.startswith("hf://"):
             request.system_prompt = settings.DEFAULT_SUMMARIZER_PROMPT
 
-        config_params = {
-            "job_name": request.name,
-            "job_id": record.id,
-            "model_path": request.model,
-            "dataset_path": dataset_s3_path,
-            "max_samples": request.max_samples,
-            "storage_path": storage_path,
-            "model_url": model_url,
-            "system_prompt": request.system_prompt,
-        }
+        # this section differs between inference and eval
+        if job_type == JobType.EVALUATION:
+            job_params = {
+                "job_id": record.id,
+                "job_name": request.name,
+                "model_path": request.model,
+                "dataset_path": dataset_s3_path,
+                "max_samples": request.max_samples,
+                "storage_path": self.storage_path,
+                "model_url": model_url,
+                "system_prompt": request.system_prompt,
+            }
+        else:
+            job_params = {
+                "job_id": record.id,
+                "job_name": request.name,
+                "model_path": request.model,
+                "dataset_path": dataset_s3_path,
+                "max_samples": request.max_samples,
+                "storage_path": self.storage_path,
+                "model_url": model_url,
+                "system_prompt": request.system_prompt,
+                "output_field": request.output_field,
+                "max_tokens": request.max_tokens,
+                "frequency_penalty": request.frequency_penalty,
+                "temperature": request.temperature,
+                "top_p": request.top_p,
+            }
+
+        return job_params
+
+    def create_job(self, request: JobEvalCreate | JobInferenceCreate) -> JobResponse:
+        """Creates a new evaluation workload to run on Ray and returns the response status."""
+        # TODO: we might want to explicitly provide a job string definition as a request
+        # parameter, so we can reuse the same config profiles across different job types
```

Comment:
Sorry, I'm just a broken record of unhelpful comments 😭 

same question as: https://github.com/mozilla-ai/lumigator/pull/328/files#r1842617684